kpi/utils/schema_extensions/response.py

```diff
@@ -194,3 +147,115 @@ def open_api_302_found(
         raise_not_found=raise_not_found,
         **kwargs,
     )
+
+
+def open_api_http_example_response(
+    name: str,
+    value: str,
+    summary: str,
+    require_auth: bool = True,
+    validate_payload: bool = True,
+    raise_access_forbidden: bool = True,
+    raise_not_found: bool = True,
+    **kwargs,
+):
+    response = {
+        (HTTP_200_OK, 'text/html'): OpenApiResponse(
+            response=str,
+            examples=[
+                OpenApiExample(
+                    name=name,
+                    value=value,
+                    summary=summary,
+                )
+            ],
+        )
+    }
+
+    return open_api_error_responses(
+        response=response,
+        require_auth=require_auth,
+        validate_payload=validate_payload,
+        raise_access_forbidden=raise_access_forbidden,
+        raise_not_found=raise_not_found,
+        media_type='text/html',
+        **kwargs,
+    )
+
+
+def open_api_error_responses(
```

Comment:
Please re-order the file ;-) open_api_error_responses > open_api_generic_response > open_api_http_example_response